### PR TITLE
Release `4.0.1`

### PR DESCRIPTION
**Pull request by delawski**
## Release Checklist

- [x] This pull request is to the `master` branch.
- [x] Release version follows [semantic versioning](https://semver.org). Does it include breaking changes?
- [x] Update changelog in `readme.txt`.
- [x] Bump version in `stream.php`.
- [x] Bump `Stable tag` in `readme.txt`.
- [x] Bump version in `classes/class-plugin.php`.
- [x] Draft a release [on GitHub](https://github.com/xwp/stream/releases/new).

Change `[ ]` to `[x]` to mark the items as done.

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -127,13 +127,40 @@ Past Contributors: fjarrett, shadyvb, chacha, westonruter, johnregan3, jacobschw
...
+- Fix: Replace `strlen` with custom callback that accepts `null` by [@delawski](https://github.com/delawski) in [#1513](https://github.com/xwp/stream/pull/1513) and [@ocean90](https://github.com/ocean90) in [#1466](https://github.com/xwp/stream/pull/1466)
+- Fix: BuddyPress Connector - check for BP dependencies before blindly assuming they're available by [@dd32](https://github.com/dd32) in [#1517](https://github.com/xwp/stream/pull/1517)
+- Fix: Adding multiple columns to the Stream table using filters HTML by [@delawski](https://github.com/delawski) in [#1519](https://github.com/xwp/stream/pull/1519)
+- Fix: Offset warning in Slack alert when there is no custom logo by [@tharsheblows](https://github.com/tharsheblows) in [#1522](https://github.com/xwp/stream/pull/1522)
+- Development: VSCode `launch.json` and update Docker troubleshooting  by [@tharsheblows](https://github.com/tharsheblows) in [#1496](https://github.com/xwp/stream/pull/1496)
+- Development: Publish Docker image on `master` only by [@tharsheblows](https://github.com/tharsheblows) in [#1505](https://github.com/xwp/stream/pull/1505)
```
Comment (tharsheblows) [inline readme.txt:149]: **🥜 nitpick**
I think take this out as it's been changed since

File: readme.txt [hunk 2]
```diff
@@ -127,13 +127,40 @@ Past Contributors: fjarrett, shadyvb, chacha, westonruter, johnregan3, jacobschw
...
 Use only `$_SERVER['REMOTE_ADDR']` as the client IP address for event logs without additional support for `X-Forwarded-For` HTTP request header value which could be spoofed. See the changelog for additional details.
 
 
 == Changelog ==
 
+= 4.0.1 - July 26, 2024 =
+
+- Fix: CLI PHP Type error by [@Soean](https://github.com/Soean) in [#1475](https://github.com/xwp/stream/pull/1475)
+- Fix: [Security] Update `select2` to `4.0.13` (by [@delawski](https://github.com/delawski) in [#1495](https://github.com/xwp/stream/pull/1495))
+- Fix: Uncaught ValueError in Gravity Forms and WordPress SEO connectors by [@tharsheblows](https://github.com/tharsheblows) in [#1508](https://github.com/xwp/stream/pull/1508)
```
Comment (tharsheblows) [inline readme.txt:141]: **🧹 chore**
Could we add a props to krokodok here? I used his PR as a starting point.

File: readme.txt [hunk 3]
```diff
@@ -127,13 +127,40 @@ Past Contributors: fjarrett, shadyvb, chacha, westonruter, johnregan3, jacobschw
 
 == Upgrade Notice ==
 
-= 4.0.0 =
+= 4.0.1 =
 
 Use only `$_SERVER['REMOTE_ADDR']` as the client IP address for event logs without additional support for `X-Forwarded-For` HTTP request header value which could be spoofed. See the changelog for additional details.
```
[the remark this answers is not in the document]
Comment (delawski) [reply readme.txt:132]: This breaking change notice was in the readme before and it refers to the 4.0.0 release. I believe we should leave it as is.

**Review by tharsheblows — DISMISSED** [review]
@delawski This looks like it should!

**Pull request merged**